<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_msgBox1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MessageBox1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kviz">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kv√≠z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nacist">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nacist obrazek</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_image">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>671</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ulozitInfo">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ulozit info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_color">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Barvy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_font">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zmena fontu</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_dialogy">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dialogy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pobocky">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>450</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vyber pobocky</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
